client/src/main/java/org/spine3/base/Commands.java

```diff
@@ -130,7 +129,7 @@ public static CommandId getId(Command command) {
             public boolean apply(@Nullable Command request) {
                 checkNotNull(request);
                 final Timestamp timestamp = getTimestamp(request);
-                return Timestamps.isAfter(timestamp, from);
+                return Timestamps.isLater(timestamp, from);
```

Comment:
Yes, but they have a method of a class, so they write:
'time.isAfter(time2)'
And we have a separate function, so to be not confusing, I wanted to change it to:
'isLater(time1, thanTime2)`